### PR TITLE
Alert for 2017-06-23.a degraded networking (fixed in 1.0.10-1)

### PR DESCRIPTION
**Pull request by bitcartel**
Alert 1004 (version 1.0.10 only) Issue #2477

### CODE REVIEW

**Comment by bitcartel** [comment]
Note that there is no need to merge this PR before sending, it can done after the event.

File: src/sendalert.cpp [hunk 1]
```diff
@@ -86,8 +86,9 @@ void ThreadSendAlert()
     //  2000 for longer invalid proof-of-work chain
     //  Higher numbers mean higher priority
     //  4000 or higher will put the RPC into safe mode
-    alert.nPriority     = 1500;
+    alert.nPriority     = 4000;
     alert.strComment    = "";
+    alert.strStatusBar  = "Your client version 1.0.10 has degraded networking behavior. Please downgrade to Zcash 1.0.9 and when available, update to the most recent version of Zcash (1.0.10-1 or later).";
     alert.strStatusBar  = "Your client is out of date and vulnerable to denial of service. Please update to the most recent version of Zcash (1.0.8-1 or later). More info at: https://z.cash/support/security/";
```
Comment (daira) [inline src/sendalert.cpp]: This overwrites the intended message with the previous one. Will fix.

**Comment by daira** [comment]
Changing milestone to 1.0.11 because this won't actually be in the 1.0.10-1 release, which has already been tagged.

**Comment by bitcartel** [comment]
ACK. I saw the alert when running 1.0.10-1 and it was put into rpc safe mode.

**Comment by bitcartel** [comment]
This page still needs updating with the new alert: https://github.com/zcash/zcash/wiki/specification#assigned-numbers

**Comment by daira** [comment]
@bitcartel do you mean 1.0.10?

**Comment by bitcartel** [comment]
ACK

**Comment by daira** [comment]
@zkbot r+

**Comment by daira** [comment]
Added alert 1004 to assigned numbers page: https://github.com/zcash/zcash/wiki/specification#mainnet-alert-codes

**Pull request merged**